build.sbt

```diff
@@ -9,6 +9,8 @@ lazy val scala213Version = "2.13.2"
 lazy val supportedScalaVersions = List(scala212Version, scala213Version)
 
 scalaVersion in ThisBuild := scala213Version
+version in ThisBuild ~= (_.replaceFirst("\\+[^+]+$", ""))
```

Comment:
I'd rather not commit that piece into `master`, if version mangling is necessary is better to be performed from the depending project's CI scripts instead

Reply:
ok